akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/JdbcProjectionImpl.scala

```diff
@@ -251,10 +251,21 @@ private[projection] class JdbcProjectionImpl[Offset, Envelope, S <: JdbcSession]
         settings) {
 
     implicit val executionContext: ExecutionContext = system.executionContext
-    override def logger: LoggingAdapter = Logging(system.classicSystem, this.getClass)
-
-    override def readPaused(): Future[Boolean] =
-      offsetStore.readManagementState(projectionId).map(_.exists(_.paused))
+    override val logger: LoggingAdapter = Logging(system.classicSystem, this.getClass)
+
+    override def readPaused(): Future[Boolean] = {
+      offsetStore
+        .readManagementState(projectionId)
+        .map(_.exists(_.paused))
+        .recover {
+          case exc =>
+            // FIXME when releasing 1.2.0 this can be removed
+            logger.info(
+              "Create the [akka_projection_management] table if you have updated to version 1.1.1 or later. See schema at https://doc.akka.io/docs/akka-projection/current/jdbc.html#schema cause: {}",
+              exc)
+            false
+        }
```

Comment:
@octonato I have implemented the fallback. It's only for this `readPaused` that is used when the projections are started. I think that is ok, since if there is some db connectivity problem the next readOffset will tell about that.

wdyt?


The management operations `pause`, `resume`, `isPaused` will still fail.

Reply:
ok, let's do that then, but then we declare 1.1.x dead and we will not backport anything